internal/datacoord/compaction_trigger.go

```diff
@@ -229,6 +229,7 @@ func (t *compactionTrigger) handleGlobalSignal(signal *compactionSignal) {
 			isSegmentHealthy(segment) &&
 			isFlush(segment) &&
 			!segment.isCompacting && // not compacting now
+			!(segment.GetState() == commonpb.SegmentState_Importing) && // not importing segment
```

Comment:
Actually the segment state will change from importing -> sealed -> flushing -> flushed for importing segments.

Considering adding another field to segmentInfo? Similar to `SaveBinlogPathsRequest.importing`
 field.